solutions/swb-ui/scripts/dev.sh

```diff
@@ -0,0 +1,10 @@
+##This script must only run by using command `STAGE=<STAGE> rushx dev` from `/swb-ui` directory, running this script directly from this directory will not work correctly as it uses relative paths.
+apiURL=$(cat ../swb-reference/src/config/${STAGE}.js | grep apiUrlOutput | awk '{print $NF}' | sed 's/\"//g' | sed 's/,//g' ) ##Get value from swb-reference/src/config/{STAGE}.js and replace all '"' and ',' with empty.
+if [ $apiURL == '']
```

Comment:
When I ran the code, I'm getting this error.
```
./scripts/dev.sh: line 3: [: https://xxxxxx.execute-api.ca-central-1.amazonaws.com/dev/: unary operator expected
```

Are you getting this error as well? Is there a way to fix it?

Reply:
seems like bash was looking for a number comparison and couldn't accept an empty space, so I changed it for command `if [[ -z $apiURL ]];` and tested it; Now its working with no errors thrown